plugins/deps-core/src/main/java/tools/forma/deps/ConfigurationType.kt

```diff
@@ -31,7 +31,8 @@ class FileSpec(
 class NameSpec(
     val name: String,
     config: ConfigurationType,
-    val transitive: Boolean = false
+    val transitive: Boolean = false,
+    val platform: Boolean = false
```

Comment:
Should we do a `PlatformSpec` instead?

Reply:
I'm not sure, because it still nameSpec with specific behavior of resolving dependencies, platform is similar with transitive from this point of view. But it was just simple way to add platform, maybe we can add one more Spec and use it

Reply:
Single type for it will be more clearly, than added overriding versions of transitive methods for passing one changing value of argument.